src/styles/main.scss

```diff
@@ -1,3 +1,65 @@
+$shift : 101px;
+
 body {
   margin: 0;
 }
+
+.calendar {
+  display: flex;
+  padding: 10px;
+  margin: auto;
+  gap: 1px;
+  width: 726px;
+  flex-wrap: wrap;
+
+  &__day {
+    display: flex;
+    box-sizing: border-box;
+    background-color: #eee;
+    border: #000 solid 1px;
+    width: 100px;
+    height: 100px;
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 30px;
+    color: #000;
+    justify-content: center;
+    align-items: center;
+    text-align: center;
+    transition: transform 0.5s ease;
```

Comment:
What about background-color? It should be changed smoothly on hover too
![image](https://github.com/mate-academy/layout_calendar/assets/96409563/be06dd09-b2a6-4e4b-b049-df4c66a19b54)